admix/interfaces/rucio_api.py

```diff
@@ -30,6 +18,21 @@
 import os
 import json
 
+def load_rucio():
```

Comment:
I made comment in main discussion, but can you explain the motivation for this?

Reply:
agree, done in e77dd0e, it wasn't linear getting travis working, was a bit too rigorous here